packages/commonwealth/package.json

```diff
@@ -226,5 +226,7 @@
     "zod": "^3.22.4",
     "zustand": "^4.3.8"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "tsx": "^4.7.2"
```

Comment:
tsx should go in the normal dependencies because we execute yarn scripts on one-off Heroku dynos. If it is I dev dependencies then tsx will be pruned during the build phase.

Reply:
Updated!